### PR TITLE
Vue test

### PR DESCRIPTION
**Pull request by NickHackman**
Fixed bug in how Vue handled different comments, since it's composed of JavaScript, HTML, and CSS sections. Added all variants to be able to handle counting properly. Removed duplicate `quotes` in Vue, as mentioned in #488 

Added Vue test

### CODE REVIEW

**Comment by NickHackman** [comment]
Initial build failed as a result of
```text
[28] Timeout was reached (download of `serde_cbor v0.11.1` 
failed to transfer more than 10 bytes in 30s)
```

**Comment by XAMPPRocky** [comment]
Thank you for your PR! However I would prefer to keep the current formatting of `languages.json` as it is more compact. If you could remove that change I would be happy to merge this in.

File: languages.json [hunk 1]
```diff
@@ -1195,8 +1195,8 @@
     },
     "Vue": {
       "name": "Vue",
-      "multi_line_comments": [["<!--", "-->"]],
-      "quotes": [["\\\"", "\\\""], ["'", "'"]],
+      "line_comment": ["//"],
+      "multi_line_comments": [["<!--", "-->"], ["/*", "*/"],
```
Comment (XAMPPRocky) [inline languages.json:1199]:
```suggestion
      "multi_line_comments": [["<!--", "-->"], ["/*", "*/"]],
```

Comment (NickHackman) [reply languages.json]: Yeah, forgot the closing `]` thanks :+1:

**Comment by NickHackman** [comment]
@XAMPPRocky the changes you've requested have been implemented :smile:

**Comment by XAMPPRocky** [comment]
Thank you again for your PR!

**Pull request merged**